docs/en/observability/aws-firehose/aws-firehose-setup.asciidoc

```diff
@@ -0,0 +1,203 @@
+| `logs-aws.cloudfront_logs-default`
+
+| https://docs.elastic.co/en/integrations/aws/ec2[EC2 (via Cloudwatch)]
+| `logs-aws.ec2_logs-default`
+
+| https://docs.elastic.co/en/integrations/aws/elb[ELB]
+| `logs-aws.elb_logs-default`
+
+| https://docs.elastic.co/en/integrations/aws/firewall[Network firewall]
+| `logs-aws.firewall_logs-default`
+
+| https://docs.elastic.co/en/integrations/aws/route53[Route 53 public DNS queries]
+| `logs-aws.route53_public_logs-default`
+
+| https://docs.elastic.co/en/integrations/aws/route53[Route 53 resolver queries]
+| `logs-aws.route53_resolver_logs-default`
+
+| https://docs.elastic.co/en/integrations/aws/s3[S3 server access]
+| `logs-aws.s3access-default`
+
+| https://docs.elastic.co/en/integrations/aws/vpcflow[VPC Flow Logs]
+| `logs-aws.vpcflow-default`
+
+| https://docs.elastic.co/en/integrations/aws/waf[WAF]
+| `logs-aws.waf-default`
+
+|===
++
+As per the https://www.elastic.co/blog/an-introduction-to-the-elastic-data-stream-naming-scheme[data stream naming conventions], the "namespace" is a user-configurable arbitrary grouping, and can be changed from `default` to fit your use case. For example, you may want to organize WAF Logs per environment into `logs-aws.waf-production` and `logs-aws.waf-qa` data streams for more granular control over rollover, retention, and security permissions.
++
+For log types not listed above, review the relevant https://docs.elastic.co/integrations/aws[integration documentation] to determine the correct *`es_datastream_name`* value.
+The data stream components can be found in the example event for each integration.
++
+[role="screenshot"]
+image::../images/firehose-integration-data-stream.png[AWS WAF integration documentation showing a sample event with the data stream components highlighted]
+
+* The *`include_cw_extracted_fields`* parameter is optional and can be set when using a CloudWatch logs subscription filter as the Firehose data source. 
+When set to `true`, extracted fields generated by the filter pattern in the subscription filter will be collected.
+Setting this parameter can add many fields into each record and _may significantly increase data volume in Elasticsearch_.
+As such, use of this parameter should be carefully considered and used only when the extracted fields are required for specific filtering and/or aggregation.
+
+* The *`include_event_original`* field is optional and _should only be used for debugging purposes_.
+When set to `true`, each log record will contain an additional field named `event.original`, which contains the raw (unprocessed) log message.
+This parameter will increase the data volume in Elasticsearch and should be used with care.
+
++
+[discrete]
+[[aws-firehose-config-destination-settings-buffer-size]]
+// <h6>
+==== Buffer size
+Elastic requires a *Buffer size* of 1MiB to avoid exceeding Elasticsearch's `http.max_content_length` setting (typically 100MB) when the buffer is uncompressed.
```

Comment:
```suggestion
Elastic requires a *Buffer size* of 1MiB to avoid exceeding the {es} `http.max_content_length` setting (typically 100MB) when the buffer is uncompressed.
```